### DISCUSSION TITLE
RestEasy Client is not working. ClassNotFoundException.

### DISCUSSION THREAD

**Discussion by simplyi** [keycloak/keycloak#29837] [Archive]
I implemented User Storage SPI and my implementation makes HTTP call to a remote Microservice using RestEasy client. 

When I deploy this implementation to Keycloak 24.0.4 and try using it, I get exception message in Keycloak server saying that: "java.lang.ClassNotFoundException: javax.ws.rs.client.ClientBuilder".

This used to work in earlier version of Keycloak but does not work in newer distribution. Adding RestEasy-JAXRS to Keycloak's "provider" folder or "lib" folder does not help. 

Has anyone encountered and resolved this issue? I wonder how? 🤔

**Comment by shawkins** [comment]
Keycloak has removed the usage of resteasy classic, which provided this functionality. Updating to resteasy reactive supported mechanisms is the way forward.

**Reply by hantsy** [reply]
Reactive is another topic.

Since Jakarta EE 9, the namespace `javax` -> `jakarta` in all Jakarta EE specs.

**Comment by simplyi** [comment]
Thank you, Shawkins! 👍🏻

**Reply by jwgmeligmeyling** [reply]
Did you ever get this to work?

Quarkus driving me insane
